templates/_asset-card-image.html

```diff
@@ -49,14 +49,14 @@
       <div class="p-section--shallow">
         <span>Tags:&nbsp;</span>
         {% for product in asset.products %}
-          <button class="p-chip is-inline is-dense">
+          <a href="/manager?tag={{product.name}}" class="p-chip is-inline is-dense">
             <span class="p-chip__value">{{ product.name }}</span>
```

Comment:
We are storing product slug in database and we try to match the exact strings here
https://github.com/canonical/assets.ubuntu.com/blob/a2ab379fc342a870c2b3f885eaa4be7c6deadb02/webapp/services.py#L56-L57
So this might only work for products that have same name and slug in lowercase